server/rpc.go

```diff
@@ -210,7 +208,7 @@ func verifySegCreds(orch Orchestrator, segCreds string) (*core.SegTranscodingMet
 		OS:         os,
 	}
 
-	if !orch.VerifySig(broadcasterAddress, string(md.Flatten()), segData.Sig) {
```

Comment:
@yondonfu I couldn't find where in the old code we used `broadcasterAddress` to sign anything, so how did this address come up as valid until now?

Reply:
It wasn't coming up as valid - an orchestrator would not have been able to transcode in on-chain mode because every segment received from a broadcaster would fail the signature verification up until now. However, in off-chain mode `orch.VerifySig` would always return `true` so an orchestrator could have still transcoded in off-chain mode.

Reply:
When testing in on-chain mode, I just commented out `orch.VerifySig` for the time being, knowing we'd eventually remove the broadcasterAddress stub